src/Test/Utilities/Shared/Metadata/ILBuilderVisualizer.cs

```diff
@@ -70,7 +69,7 @@ private static List<HandlerSpan> GetHandlerSpans(ImmutableArray<Cci.ExceptionHan
             {
                 HandlerSpan span;
 
-                if (region.HandlerKind == ExceptionRegionKind.Filter)
+                if (region.HandlerKind == System.Reflection.Metadata.ExceptionRegionKind.Filter)
```

Comment:
It looks like the only changes in this file is the removal of using and a fix-ups caused by that. If that is the case. Please revert the change, at least exclude it from this PR as it looks unrelated to the issue being addressed.